frontend/src/components/EventPage/EventPage.js

```diff
@@ -136,6 +137,7 @@ const EventPage = ({ eventIndex }) => {
   const { eventImages, fetchEventImage } = useEventImagesContext();
   const { setActivePage } = usePageContext();
   const { modalsIsOpen, openModal, closeModal } = useModalHandler(MODALS);
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
instead of this `useState` usage, use `const { modalsIsOpen, openModal, closeModal } = useModalHandler(MODALS);`.
To do it, it is needed to add a new MODALS type, like this ->
```
const MODALS = {
  editTitle: "editTitle",
  editImage: "editImage",
  generateTickets: "generateTickets",
  publishErrorModal: "publishErrorModal",
};
```
and in all places where you are using `isOpen` and `setIsOpen` you should use `modalIsOpen[MODALS[publishErrorModal]`, `openModal(MODALS[publishErrorModal])`, `closeModal(MODALS[publishErrorModal])`

Reply:
updated